.travis.yml

```diff
@@ -75,6 +73,7 @@ matrix:
         # latest Python version.  The duration of the slowest 50 tests
         # will also be listed.
         - env: SETUP_CMD='test --coverage --remote-data=astropy -a "--durations=50"'
+               NUMPY_VERSION=1.14
```

Comment:
It looks like this test is already running with `numpy 1.14.5` as `NUMPY_VERSION=stable` (e.g. https://travis-ci.org/astropy/photutils/jobs/424925578).  Why do we need to specify the numpy version here?

Reply:
Ah, OK.  I didn't realize numpy 1.15 came out today.  Thanks!

Reply:
yes, it's out for a while, it's just taken a while to get ci-helpers updated.